test/e2e/lib/components/abstract-editor-component.js

```diff
@@ -33,6 +33,20 @@ export default class AbstractEditorComponent extends AsyncBaseContainer {
 		await driverHelper.clickWhenClickable( this.driver, patternItemLocator );
 	}
 
+	async openBlockInserterAndSearch( searchTerm ) {
+		await this.runInCanvas( async () => {
+			await driverHelper.scrollIntoView(
+				this.driver,
+				By.css( '.block-editor-writing-flow' ),
```

Comment:
It looks like this class is no longer available in the Site Editor.  I think we can replace it with `.editor-styles-wrapper` and have it work in both editors?  If not we may want to revert the DRY and update the site editor one to use the styles wrapper.    https://github.com/Automattic/wp-calypso/pull/55245